indexer-app/src/cmr/indexer/data/collection_granule_aggregation_cache.clj

```diff
@@ -93,10 +93,10 @@
                              (get-in bucket [:max-temporal :value]))
                  some-with-no-end (> (get-in bucket [:no-end-date :doc_count]) 0)]]
        [concept-id
-        {:granule-start-date earliest-start
+        {:granule-start-date-stored earliest-start
```

Comment:
Just an FYI.

We try not to expose the Elasticsearch field implementation beyond the `cmr.search.data.query-to-elastic` namespace. This means calling:

```(q2e/query-field->elastic-field :granule-start-date :collection)```
to pull back the value. The CMR code base is really inconsistent, and I'm seeing the field names leak out in a bunch of namespaces probably because it's much easier to understand, so I'm fine with this PR.

Reply:
Thank you!